apps/rehabstod/src/components/SettingsDialog/SettingsDialog.tsx

```diff
@@ -0,0 +1,84 @@
+import { IDSButton, IDSButtonGroup, IDSDialog } from '@frontend/ids-react-ts'
+import { ReactNode, useState } from 'react'
+import { NumberInput } from '../Form/NumberInput'
+import { useUpdateUserPreferencesMutation } from '../../store/api'
+import { UserPreferences } from '../../schemas'
+
+export function SettingsDialog({
+  preferences,
+  children,
+  show,
+  onClose,
+}: {
+  preferences: UserPreferences | undefined
+  children: ReactNode
+  show: string
+  onClose: () => void
+}) {
+  const [UpdateUserPreferences] = useUpdateUserPreferencesMutation()
+  const [savedPreferences, setSavedPreferences] = useState<UserPreferences | undefined>(preferences)
+
+  if (!savedPreferences) {
+    return null
+  }
+
+  const onSave = () => {
+    if (savedPreferences) {
+      UpdateUserPreferences(savedPreferences)
+    }
+    onClose()
+  }
+
+  return (
+    <IDSDialog dismissible headline="Inställningar" show={show}>
+      {children}
+      <div>
+        <h2 className="ids-heading-4">Visa nyligen avslutade sjukfall</h2>
+        <p>
+          Välj maximalt antal dagar som får ha passerat efter ett sjukfalls slutdatum för att sjukfallet ska visas upp i sjukfallstabellen.
+          Med denna funktion kan du bevaka de sjukfall som är nyligen avslutade. Välj 0-14 dagar.
+        </p>
+        <NumberInput
+          label="Max antal dagar sedan avslut"
+          onChange={(event) =>
+            setSavedPreferences({
+              ...savedPreferences,
+              maxAntalDagarSedanSjukfallAvslut: event.currentTarget.value,
+            })
+          }
+          id="daysAfterSickLeaveEnd"
+          value={savedPreferences.maxAntalDagarSedanSjukfallAvslut ? Number(savedPreferences.maxAntalDagarSedanSjukfallAvslut) : 0}
```

Comment:
Seems like I cant remove the 0 in this component by using backspace-key. To change it I need to mark it and then type the desired number:

![image](https://user-images.githubusercontent.com/57347249/233297683-6a1f2f7c-f79e-4932-9051-a7fef09d6666.png)